tests/installation/bootloader_s390.pm

```diff
@@ -89,7 +89,12 @@ sub prepare_parmfile {
         $params .= " info=" . create_infofile("install: $instsrc");
     }
     else {
-        $params .= " install=" . $instsrc . $repo . " ";
+        if (!get_var('AGAMA_TEST')) { 
```

Comment:
would make sense to have all the code together to run here without `!` and reverse the branches of the condition.

Reply:
updated.